src/clarity-angular/utils/conditional/if-open.directive.ts

```diff
@@ -68,7 +70,13 @@ export class IfOpenDirective implements OnInit {
     }
 
     constructor( private ifOpenService: IfOpenService, private template: TemplateRef<any>,
```

Comment:
For the conditional directives, subscribing to them inside `OnInit` is too late, as the value whose change we are subscribing to can be set earlier as input in the component and the directive needs to react to that value. It just depends on the scenario. Copy/pasting from our thread outside of GitHub:

OnInit is an excellent place for:
- setup that needs the inputs to be ready
- heavy initial action that you don't want to trigger when you just instantiate the component manually in tests (an HTTP request, for instance)